ext/java/org/jruby/ext/nkf/Command.java

```diff
@@ -0,0 +1,59 @@
+/***** BEGIN LICENSE BLOCK *****
+ * Version: EPL 2.0/GPL 2.0/LGPL 2.1
```

Comment:
@headius Should we handle `GPL 2.0` for JRuby version at https://github.com/ruby/nkf/pull/15/files#diff-981027bf28c71984fb019c36c09b0d12b2c84c185d306c6caede7ccf7a76d4a3R21 ?

Reply:
We can just drop GPL altogether. EPL (business-friendly) + LGPL (GPL-compatible) should cover every case we need. Would that be ok?

Reply:
(I'm not lawyer) EPL and LGPL are ok to me. Can you handle them under https://github.com/ruby/nkf/pull/15/files#diff-981027bf28c71984fb019c36c09b0d12b2c84c185d306c6caede7ccf7a76d4a3R34?

Reply:
I have removed GPL (we know it scares people and have considered removing it from JRuby too) and added the additional JRuby licenses to the java gem.